src/ViewRendererBundle/objects/helper/TPkgViewRendererLessCompiler.php

```diff
@@ -253,26 +253,11 @@ public function writeCssFileForPortal($css, $portal)
     }
 
     /**
-     * returns the URL path to the less directory.
-     *
-     * @return string
+     * @return string - the path part of the URL to the less directory
      */
-    protected function getLessDirUrlPath()
+    protected function getLessDirUrlPath(): string
```

Comment:
Signature changes are not allowed in fix releases (in general I'd only use return value declarations in new code).

Reply:
ok